### ISSUE TITLE
Use as a base or Sdk?

### ISSUE THREAD

**Issue by FilBot3** [fastrizwaan/flatpak-wine#5]
Is it possible to use this flatpak as a base or sdk so as to package a Windows application in a Flatpak?

**Comment by fastrizwaan** [comment]
Yes,

If you want simpler way, Winepak is just a script which bundles directplay/installed application directory and wine together, which can be used https://github.com/fastrizwaan/winepak-x86_64

or you can have this with an installer, see this https://github.com/johnramsden/pathofexile-flatpak/blob/master/ca.johnramsden.pathofexile.json

I've created a script, to create desktop shortcuts for exe files, perhaps I'll add another script to create a game bottle which can be run with this flatpak.

**Comment by FilBot3** [comment]
Awesome, I'll take a look. I guess this issue can be closed. I'll update with anything I figure out as well. I'm looking to add an application that is supposed to have Linux clients soon, but the developer hasn't and with as many times as I'm reinstalling my Host OS, having a flatpak with that tool on it already ready to go, would be spiffy.

**Comment by FilBot3** [comment]
Reminds me of this site: https://www.winepak.org/ but those repositories haven't seen any love since 2018.

**Comment by fastrizwaan** [comment]
Actually, I just wanted to create wine bundles to save my apps and games, usually, .wine get corrupted/messed up with testing various programs. I saw with flatpak-linux script (make-flatpak.sh) https://pirateproxy.ltda/search.php?q=flatpak-linux&all=on&search=Pirate+Search&page=0&orderby=, but it depended on PlayOnLinux for wine. Flatpak has different libraries. So to make flatpak sdk compatible wine build and to create wine bundles, I've created this flatpak.

The issues I faced were, wine was continuously updating in distro causing apps to break. .wine directory gets messy with various setup files and installations. So, needed a clean wine bottles. Lutris, Bottles are not working as expected.

Now that I've made wine wow64 with flatpak 20.08, it will be supported at least for 5 years. so wine apps inside a bundle will be easier to work with. also this provides wine to centos like distros where wine 32 bit is not provided.
